pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -179,6 +179,11 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 	output, err := p.ec2api.CreateLaunchTemplateWithContext(ctx, &ec2.CreateLaunchTemplateInput{
 		LaunchTemplateName: aws.String(launchTemplateName(options)),
 		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
+			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
+				Ebs: &ec2.LaunchTemplateEbsBlockDeviceRequest{
+					Encrypted: aws.Bool(true),
+				},
+			}},
```

Comment:
Yep, and so will VolumeSize 

```suggestion
			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
				Ebs: &ec2.LaunchTemplateEbsBlockDeviceRequest{
					Encrypted:  aws.Bool(true),
					VolumeSize: aws.Int64(20),
				},
				DeviceName: aws.String("/dev/sdf"),
			}},
```

Reply:
strange, I'm still seeing errors:

```
2022-01-29T00:46:17.712Z	ERROR	controller.provisioning	Could not launch node, launching instances, with fleet error(s), InvalidBlockDeviceMapping: Missing device name	{"commit": "0517b4a", "provisioner": "default"}
```

Reply:
oh i'm understanding more now.  previous runs of Karpenter has already built the launchTemplate, so my update logic wasn't getting triggered.  deleting all existing Karpenter launch templates results in a new template getting generated correctly (encrypted, named correctly, and 20gb).